apps/users/tests/test_views.py

```diff
@@ -518,16 +522,50 @@ def test_ok_redirects(self):
         r = self.client.get(self.url + '?to=/de/firefox/', follow=True)
         self.assert3xx(r, '/de/firefox/')
 
-    def test_bad_redirects(self):
+    def test_bad_redirect_other_domain(self):
+        r = self.client.post(
+            self.url + '?to=https://example.com/this/is/bad',
+            self.data, follow=True)
+        self.assert3xx(r, '/en-US/firefox/')
```

Comment:
are you sure this is a successful login? It would be good to make an assertion about that (if possible).